packages/editor/src/components/parts/program/event/ProgramStartPart.test.tsx

```diff
@@ -17,7 +17,10 @@ describe('StartPart', () => {
 
   test('empty data', async () => {
     renderPart();
+    await SelectUtil.assertEmpty();
+    //await SelectUtil.assertOptionsCount(1);

```

Comment:
comment?

Reply:
Yes I have left this line as a comment as I don't know yet what is the correct value for assertOptionsCount as I am currently not getting any data from the backend